app/client/src/layout/SideMenu.jsx

```diff
@@ -48,7 +48,7 @@ const SideMenu = (props) => {
   }
 
   return (
-    <div id="content">
+    <div id="content" style={{ zIndex: 500 }}>
```

Comment:
We actually aren't using this file anymore so we could probably delete it entirely.